drivers/SmartThings/harman-luxury/profiles/jbl-ma-avr-series.yaml

```diff
@@ -1,4 +1,4 @@
-name: maX10
+name: jbl-ma-avr-series
```

Comment:
I don't know that we allow for renaming profiles in production as this will be seen as deleting a profile and creating a new one and will orphan all existing devices.  @greens or @calebvarberg do you know if this is allowable?

Reply:
Well, the device is not in the market yet. So it won't affect any end-user.

Reply:
But it has been published to production, which is where the restriction lies.  If this is not allowed by the platform it will require special privileges to change if that's even possible.  We'll see if this is possible given that the driver has been released.

Reply:
Okay I understand. If the displayed name in the menu I shown in the picture above could be changed, this PR can be scraped.
Harman Luxury want to show under "Device Name" the device model (e.g. `JBL MA510`), but if it's impossible, as there are 4 devices using this profile, the series name will be sufficient (i.e. `JBL MA AVR Series`)